src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -167,5 +168,32 @@ private static string GetFilePath(string testFileName)
         {
             return Path.Combine(Directory.GetCurrentDirectory(), "Tests", testFileName);
         }
+
+        [DataTestMethod]
+        [DataRow(ValidTemplateConstants.PassingTest, 0, DisplayName = "Valid Template")]
+        [DataRow(ValidTemplateConstants.CaseSensitivity, 1, DisplayName = "Unexpected capitalization in schema")]
+        [DataRow(ValidTemplateConstants.DifferentSchemaDepths, 1, DisplayName = "Two schemas, different depths, valid schema last")]
```

Comment:
These tests are misleading.  In both of these cases, `IsValidTemplate` should (and likely does) return `true`.  The exit code returned by analyze-template would be a result of the test text not being a valid template, and is not related to the schema, which is what this test should be validating.

For these cases, I suggest making the text a valid template that results in a 0 exit code, so it demonstrates that `IsValidTemplate` gave the correct results.

Additionally, the description "Unexpected capitalization in schema" makes it sound like this scenario isn't allowed, but the logic does allow for it by design.  Can you rephrase so it's clear that this is a valid scenario?